index.js

```diff
@@ -212,6 +218,7 @@ exports.sendMoney = async function (options) {
 };
 
 exports.stake = async function (options) {
+    if(options.networkId === 'default' | 'testnet') await checkCredentials(options.accountId, options.networkId, options.keyStore);
```

Comment:
this should be `||`

Reply:
removed this if statement as it is now checking in the function itself